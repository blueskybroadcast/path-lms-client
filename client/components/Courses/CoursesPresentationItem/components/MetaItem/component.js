import React from 'react';
import PropTypes from 'prop-types';

const MetaItem = ({ count, icon }) => (
  <li>
    <i className={`icon icon-${icon}`} />
    <span>{count}</span>
  </li>
);

MetaItem.propTypes = {
  count: PropTypes.number.isRequired,
  icon: PropTypes.string.isRequired
};

export default MetaItem;
