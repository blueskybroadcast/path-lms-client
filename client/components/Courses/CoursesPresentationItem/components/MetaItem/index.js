import MetaItem from './component';

export default MetaItem;
