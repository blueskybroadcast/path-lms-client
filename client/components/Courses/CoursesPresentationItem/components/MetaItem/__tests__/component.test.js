import React from 'react';
import { shallow } from 'enzyme';
import MetaItem from '../component';

describe('Meta item', () => {
  test('should render MetaItem correctly', () => {
    const wrapper = shallow(
      <MetaItem
        count={5}
        icon="name"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
