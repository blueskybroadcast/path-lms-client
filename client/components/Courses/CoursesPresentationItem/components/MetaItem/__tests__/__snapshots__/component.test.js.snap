// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meta item should render MetaItem correctly 1`] = `
<li>
  <i
    className="icon icon-name"
  />
  <span>
    5
  </span>
</li>
`;
